src/snowflake/snowpark/session.py

```diff
@@ -1120,7 +1160,7 @@ def _upload_unsupported_packages(
                 self._tmpdir_handler.cleanup()
             raise RuntimeError(
                 f"Unable to auto-upload packages: {packages}, Error: {e} | NOTE: Alternatively, you can find the "
-                f"directory of these packages and add it via session.add_import(). See details at "
+                f"directory of these packages and add it via Session.add_import. See details at "
```

Comment:
`session.add_import` here is formatted differently from the in on line 990. can you make them consistent